v2/session/advanced-customizations/localstorage-example/guide.mdx

```diff
@@ -0,0 +1,467 @@
+            antiCsrf: "NONE",
+            override: {
+                functions: (origImpl) => {
+                    return {
+                        ...origImpl,
+
+                        createNewSession: async (input) => {
+                            const session = await origImpl.createNewSession(input);
+
+                            updateHeaders(session.res);
+
+                            return session;
+                        },
+
+                        refreshSession: async (input) => {
+                            const session = await origImpl.refreshSession(input);
+
+                            updateHeaders(session.res);
+
+                            return session;
+                        },
+
+                        getSession: async (input) => {
+                            const stCookies = input.req.headers["st-cookie"];
+
+                            if (stCookies) {
+                                input.req.headers["cookie"] = input.req.headers["st-cookie"];
+                            }
+                            const res = origImpl.getSession(input);
+
+                            updateHeaders(input.res);
+
+                            return res;
+                        },
+                    };
+                },
+
+                apis: (origImpl) => {
+                    return {
+                        ...origImpl,
+
+                        signOutPOST: async (input) => {
+                            await origImpl.signOutPOST(input);
+
+                            updateHeaders(input.options.res);
+                        },
+
+                        verifySession: async (input) => {
+                            const stCookies = input.options.req.headers["st-cookie"];
+
+                            if (stCookies) {
+                                input.options.req.headers["cookie"] = input.options.req.headers["st-cookie"];
+                            }
+
+                            await origImpl.verifySession(input);
+
+                            updateHeaders(input.options.res);
+                        },
+
+                        refreshPOST: async (input) => {
+                            const stCookies = input.options.req.headers["st-cookie"];
+
+                            if (stCookies) {
+                                input.options.req.headers["cookie"] = input.options.req.headers["st-cookie"];
+                            }
+
+                            await origImpl.refreshPOST(input);
+
+                            updateHeaders(input.options.res);
+                        },
+                    };
+                },
+            },
+        }),
+        //highlight-end
+    ],
+});
+```
+</TabItem>
+</BackendSDKTabs>
+
+## Frontend
+
+### 1. Adding a helper to store cookies
+First, we need to add a helper function that will process the custom header and store the cookies in the proper format.
+
+<FrontendSDKTabs>
```

Comment:
Tabs are not being displayed correctly in the docs. Anyway, you can link to the code on github instead of pasting it here.

(Also applicable to the sections below)